modules/oadp-deleting-cluster-resources-following-success.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-cluster-resources-following-success_{context}"]
+= Deleting cluster resources following a successful backup and restore using Data Mover
+
+You can delete any `volumeSnapshotBackup/volumeSnapshotRestore` CRs that remain in your application namespace after a successful backup and restore in which you used Data Mover.
+
+.Procedure
+
+. Delete cluster resources that remain after a backup in which you use Data Mover:
```

Comment:
"in which" sounds really stiff/proper. Could you use "where you used Data Mover" throughout this file?

```suggestion
. Delete cluster resources that remain after a backup where you used Data Mover:
```